templates/test/infopanel.html.ep

```diff
@@ -96,5 +96,10 @@
                         </span>
                     </div>
                 % }
+                % if ($job && $job->group && $job->group->description && length($job->group->description)) {
+                    <div id="group-description">
+                            %= $job->group->description 
```

Comment:
Well, then I doubt this is what we want. We were looking for the "testsuite description" as in https://github.com/os-autoinst/openQA/pull/1954/files , not the "job group description" :D

Reply:
Oh sorry :) I've changed group to scenario - please have a look at new commit